### ISSUE TITLE
SLIs/SLOs for cluster limits tests

### ISSUE THREAD

**Issue by chaitanyaenr** [openshift-scale/workloads#138]
The cluster limits tests today validate if we are able to reach the desired limit without much degradation in performance mainly around control plane stability and resource usage. We need to take into account the defined SLIs/SLOs in upstream Kubernetes when validating and pushing the limits. API server latency is one of the important things to consider to see how the server is responding when there are a large number of objects running on a large scale cluster. The tests need to take them into account for pass/fail.

Reference: https://github.com/kubernetes/community/blob/master/sig-scalability/slos/slos.md

**Comment by chaitanyaenr** [comment]
cc: @jtaleric @mffiedler

**Comment by jtaleric** [comment]
@chaitanyaenr The idea here is to implement the Official k8s SLI/SLOs -- not the WIP ones?

**Comment by chaitanyaenr** [comment]
@jtaleric Right, the idea is to find out the maximums with Official k8s SLI/SLOs taken into consideration and pass/fail the tests if the Official k8s SLI/SLOs are not met.
